odoo/addons/base/models/assetsbundle.py

```diff
@@ -322,20 +361,79 @@ def save_attachment(self, type, content):
 
         return attachment
 
-    def js(self):
-        attachments = self.get_attachments('js')
-        if not attachments:
+    def js(self, is_minified=True):
+        extension = 'min.js' if is_minified else 'js'
+        if is_minified:
+            attachments = self.get_attachments(extension)
+        else:
+            attachments = self.get_attachments(extension, version_mode="NONE")
```

Comment:
```suggestion
        attachments = self.get_attachments(extension, version_mode='' if is_minified else 'NONE')
```
Alternatively, put the `extension` stuff inside these statements rather than as a separate expression.